src/acl/acl.ts

```diff
@@ -0,0 +1,132 @@
+import { AccessControl } from "accesscontrol";
+import { UserRoleEnum } from "../generated/UserRole";
+
+// Resource constants
+export const ADMINISTRATION_RESOURCE = "administration";
```

Comment:
what's the difference between administration and organization ?

Reply:
- `administration` is a public administration from IndicePA, ie. an administration that never started a registration process
- `organization` is a public administration from IO, ie. an administration whose registration process already started

Reply:
can we add a comment ?